sources/packages/backend/apps/db-migrations/src/sql/SupportingUsers/Add-identifiable-user-columns.sql

```diff
@@ -0,0 +1,33 @@
+-- Supporting users new columns to alow specific user identification.
+ALTER TABLE
+    sims.supporting_users
+ADD
+    COLUMN is_able_to_report boolean NOT NULL DEFAULT TRUE,
+ADD
+    COLUMN full_name VARCHAR(250),
```

Comment:
Will it be useful to add a conditional NOT NULL constraint on full_name for supporting user type `Parent`?

Reply:
We would need to include the validation for a program year together because it will not be mandatory for previous program years.
The plane is doing such validation at the application level.

Reply:
ok. I was thinking from a production perspective, where we can write an update  to put a dummy fullname to all supporting user type = Parent and then set constraint. Btw, it is not a blocker.